WordPress/src/test/java/org/wordpress/android/ui/sitecreation/segments/NewSiteCreationSegmentsViewModelTest.kt

```diff
@@ -57,10 +59,10 @@ private val SECOND_MODEL =
                 SECOND_MODEL_SEGMENT_ID
         )
 
-private val PROGRESS_STATE = UiState(false, true, listOf(HeaderUiState, ProgressUiState))
-private val ERROR_STATE = UiState(true, false, emptyList())
-private val HEADER_AND_FIRST_ITEM_STATE = UiState(
-        false, true,
+private val PROGRESS_STATE = SegmentsContentUiState(listOf(HeaderUiState, ProgressUiState))
+private val GENERIC_ERROR_STATE = SegmentsErrorUiState.SegmentsGenericErrorUiState
```

Comment:
I am not sure if there is any value having these values anymore. Can we just use `SegmentsGenericErrorUiState` in place of `GENERIC_ERROR_STATE` since it's an object? Also, same thing for `NO_CONNECTION_ERROR_STATE`.

P.S: The only value I see for keeping it this way is to avoid having to change it if we need to make a change to the structure. In that case, I think it's better to force ourselves to make the change in the tests explicitly to make sure we think about its implications. Also, as you know by now, I am not a fan of having unused code or code that might be useful in the future in a code base.

Reply:
I thought it'd be more readable if we kept everything written in upper_case. However, it doesn't really matter. I've removed them;).